src/resources/postcss/resets/_twentynineteen.pcss

```diff
@@ -1,3 +1,21 @@
-.tribe-theme-twentynineteen {
+.tribe-theme-twentynineteen .tribe-common {
```

Comment:
Nice, I like the idea of a theme specific (more organized) reset per theme, seems helpful and more clear.